src/integration-tests/src/secret_manager/openapi.rs

```diff
@@ -39,22 +39,20 @@ pub async fn run(tracing: bool) -> Result<()> {
 
     println!("\nTesting create_secret()");
     let create = client
-        .create_secret(
-            smo::model::CreateSecretRequest::default()
-                .set_project(&project_id)
-                .set_secret_id(&secret_id)
-                .set_request_body(
-                    smo::model::Secret::default()
-                        .set_replication(
-                            smo::model::Replication::default()
-                                .set_automatic(smo::model::Automatic::default()),
-                        )
-                        .set_labels(
-                            [("integration-test", "true")]
-                                .map(|(k, v)| (k.to_string(), v.to_string())),
-                        ),
+        .create_secret()
+        .set_project(&project_id)
+        .set_secret_id(&secret_id)
+        .set_request_body(
+            smo::model::Secret::default()
```

Comment:
Nothing to do with the PR, but I wonder if all these messages should have a new() method that just calls default. I like fewer characters personally.

Reply:
Ack.  I tend to agree.

Reply:
#478 to track that work.